pkg/destroy/alibabacloud/alibabacloud.go

```diff
@@ -347,7 +347,7 @@ func (o *ClusterUninstaller) deleteResourceGroup(logger logrus.FieldLogger) (err
 		2*time.Second,
 		1*time.Minute,
 		func() (bool, error) {
-			response, err := o.listResourceGroups()
+			response, err := o.listResourceGroups(resourceGroupName)
```

Comment:
With this change, the subsequent loop through the response list can just be changed to a check for `response.TotalCount == 0`.

Reply:
This is a fuzzy search, so can't judge whether it exists by response.TotalCount. For example, Installer creates `test-xxx-rg`, then user manually creates `test-xxx-rg-2`, which results in `response.TotalCount == 0` is false